fastlane/Fastfile

```diff
@@ -86,3 +156,19 @@ def analyze_api_tester
     sh("flutter", "analyze")
   end
 end
+
+def get_phc_version
+  android_build_gradle_path = '../android/build.gradle'
+  android_build_gradle_contents = File.read(android_build_gradle_path)
+  android_phc_version = android_build_gradle_contents.match("ext.common_version = '(.*)'").captures[0]
+  UI.user_error!("Android PHC version not found.") if android_phc_version.nil?
+
+  ios_podspec_path = '../ios/purchases_flutter.podspec'
+  ios_podspec_contents = File.read(ios_podspec_path)
+  ios_phc_version = ios_podspec_contents.match("s.dependency 'PurchasesHybridCommon', '(.*)'").captures[0]
+  UI.user_error!("iOS PHC version not found.") if ios_phc_version.nil?
+
+  UI.user_error!("Android and iOS PHC versions don't match. Please make sure they match.") if android_phc_version != ios_phc_version
```

Comment:
This is a new validation that will happen during the `bump` lane. I think it's useful. That said, I'm not sure if we ever want to have different versions of PHC in iOS and Android. If so, we probably need to figure out what we want to put in the `VERSIONS.md` file.